src/modules/accounting/accounting.py

```diff
@@ -122,15 +122,13 @@ def is_extra_data_submitted(self, blockstamp: BlockStamp) -> bool:
         return processing_state.extra_data_submitted
 
     def is_reporting_allowed(self, blockstamp: ReferenceBlockStamp) -> bool:
```

Comment:
Maybe do this check in Accounting module?
In `def execute_module`?

Reply:
why? we need to collect maximum report data as we can